packages/medusa/src/types/product-category.ts

```diff
@@ -0,0 +1,26 @@
+import { Transform } from "class-transformer"
+import { IsNotEmpty, IsOptional, IsString, IsBoolean } from "class-validator"
+
+export type CreateProductCategoryInput = {
+  name: string
+  handle: string
```

Comment:
thought(non-blocking): We could make handle optional, and when it's not included, we kebab-case the name in a `beforeInsert` trigger. See the product model for reference.

Reply:
That is much better!